Update funding

loftusa opened the pull request:
do not merge yet.
Making PR to have public access to Netlify on push.

loftusa committed:
add skeleton funding page

loftusa committed:
pre rough draft for page, acknowledgements of all papers in temporarily

loftusa committed:
Merge branch 'deploy' into update-funding

loftusa committed:
finish going through arxiv papers

loftusa committed:
update formatting

falkben commented:
FYI, netlify settings changed and prior build previews are gone.  make a new commit for a new build preview.

loftusa committed:
Merge branch 'deploy' into update-funding

loftusa committed:
good enough draft of funding page to push to website. Still need sentences on many grants.

loftusa commented:
ok, should be good to merge.
I'm not particularly satisfied with it right now -- I still need sentences for most of the grants, and there isn't much meat

but I think it's decent enough to go up, and we can make improve iteratively

falkben commented on content/about/funding.md:
```diff
@@ -0,0 +1,94 @@
+
+This page lists currently active funding for our projects
+
+#### Project Title: Synaptomes of Mouse and Man
```
remove "Project Title" from each of the headings.

loftusa replied:
Will do

loftusa replied:
done

falkben commented on content/about/funding.md:
```diff
@@ -0,0 +1,94 @@
+This page lists currently active funding for our projects
+
+#### Project Title: Synaptomes of Mouse and Man
+**R01NS092474 (Smith)**: 9/30/2014 – 6/30/2019  
```
we should probably add a link to each of the funding pages for each of these grants.

for instance, for synaptomes of mouse and man, http://grantome.com/grant/NIH/R01-NS092474-01

note, the synaptomes grant is about to end.

loftusa replied:
Cool, will do. I can add a hyperlink into either the grant title or the project title.

falkben replied:
not urgent

loftusa replied:
ok, to do later then. Will add to the issue.

falkben reviewed (changes requested):
Nice.

The PDF you are replacing with this has a "pending" category, which this page does not.  Are we adding that?

I think we might want a "past" category (or separate page).  See Slack #website discussion for that.

loftusa committed:
Merge branch 'deploy' into update-funding

falkben commented:
Mobile nav still has a link to old funding page

falkben commented on partials/about_nav.html:
```diff
@@ -14,7 +14,6 @@
                 <a href="{{item.url.path}}"><h5>{{item.title}}</h5></a></li>
             </li>
         {% endfor %}
-        <li><a href="{{g.static('/source/docs/CP_Vogelstein.pdf').url.path}}" target="_blank" rel="noopener"><h5>Funding</h5></a></li>
```
this line delete: https://github.com/neurodata/neurodata.io/pull/245/files#diff-dad88da953fd09c2c8b366b95e4c44b3R24

loftusa replied:
done

loftusa committed:
remove project titles, fix mobile nav double-funding bug

loftusa commented:
ok @falkben !

- removed "Project Title:"s
- per @jovo , did not add "overlap" or "pending" information
- fixed mobile nav bug

for future commit:
- did not add "past" information (not sure what the verdict is on that? @jovo), but could do so.
- link to funding pages on grantome
- need rest of sentences
- add neurodata logo or something to make it look nicer
- potentially add link to CP_vogelstein.pdf somewhere if people need extra information

CP_vogelstein.pdf is still in `source/docs` in case we need it

falkben reviewed (approved):
Thanks looks much better
